fastlane/Fastfile

```diff
@@ -99,7 +99,6 @@ platform :ios do
     version_number = current_version_number
     check_no_git_tag_exists(version_number)
     check_pods
-    carthage_archive
```

Comment:
this one is here to make sure that we _can_ create a carthage archive. The contents get deployed with releases, so that carthage can use it by default instead of re-compiling whenever someone does `carthage update Purchases`

Reply:
✅